tests/ovirt/vm_validation_test.go

```diff
@@ -80,7 +80,6 @@ var _ = Describe("VM validation ", func() {
 		table.Entry("unsupported timezone", vms.UnsupportedTimezoneVmID, "timezone-template.xml", map[string]string{"@TIMEZONE": "America/New_York"}),
 		table.Entry("unsupported s390x architecture", vms.UnsupportedArchitectureVmID, "architecture-template.xml", map[string]string{"@ARCH": "s390x"}),
 		table.Entry("USB enabled", vms.UsbEnabledVmID, "usb-template.xml", map[string]string{"@ENABLED": "true"}),
-		table.Entry("unsupported i440fx_sea_bios BIOS type", vms.UnsupportedBiosTypeVmID, "bios-type-template.xml", map[string]string{"@BIOSTYPE": "i440fx_sea_bios"}),
```

Comment:
Please add a test case for i440fx_sea_bios to https://github.com/kubevirt/vm-import-operator/blob/4123ab618dec72f41bbba540bf19d6e7f1c6b151/tests/ovirt/various_vm_configurations_test.go#L96